package.json

```diff
@@ -46,7 +46,7 @@
   "dependencies": {
     "@matrixai/async-init": "^1.8.2",
     "@matrixai/async-locks": "^3.1.2",
```

Comment:
Is this dependency directly needed? If not, we can remove.

Reply:
Are you reffering to locks or both init and locks? The `Lock` is still used by FD operations for locking cursor updates.

Reply:
Just `async-locks`. I see ok.